ui/lib/pki/addon/components/page/pki-tidy-status.hbs

```diff
@@ -28,11 +28,11 @@
     @color={{this.tidyStateAlertBanner.color}}
     @icon={{this.tidyStateAlertBanner.icon}}
     class="has-top-margin-m"
-    data-test-hds-alert
+    data-test-tidy-status-alert
```

Comment:
trying to make selectors more component-relevant so debugging is easier in the future